lib/std/child_process.zig

```diff
@@ -491,6 +538,21 @@ pub const ChildProcess = struct {
             stderr.close();
             self.stderr = null;
         }
+        if (self.extra_streams) |extra_streams| {
+            for (extra_streams) |*extra| {
+                if (extra.*.direction == .parent_to_child) {
+                    if (extra.*.output) |*outpipe| {
+                        outpipe.close();
+                        extra.*.output = null;
+                    }
+                } else {
+                    if (extra.*.input) |*inpipe| {
+                        inpipe.close();
+                        extra.*.input = null;
+                    }
+                }
```

Comment:
No need for the `.*` most of the time.
Also you can use a `switch` here.
```suggestion
                switch (extra.direction) {
                    .parent_to_child => if (extra.output) |*outpipe| {
                        outpipe.close();
                        extra.output = null;
                    },
                    .child_to_parent => if (extra.input) |*inpipe| {
                        inpipe.close();
                        extra.input = null;
                    },
                }
```

Reply:
I used switches and `.` instead of `.*.` everywhere to increase readability and make future regressions less likely.